dataline-scheduler/src/main/java/io/dataline/scheduler/ScheduleJobPredicate.java

```diff
@@ -62,9 +62,7 @@ private boolean shouldSchedule(Optional<Job> previousJobOptional, boolean timeFo
       case CANCELLED:
       case COMPLETED:
         return timeForJobNewJob;
-      // todo (cgardens) - add max retry concept
       case FAILED:
```

Comment:
scheduler no longer in charge or scheduling failed jobs. offloaded to retryer. now it just ignores it.